File: mocks/oppgave-mock/src/main/java/no/nav/pensjon/vtp/mocks/oppgave/OppgaveInternalAliveMockImpl.java
package no.nav.pensjon.vtp.mocks.oppgave;

import io.swagger.annotations.Api;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@Api(tags = "Oppgave Mock")
@RequestMapping("oppgave/api/v1/internal/alive")
public class OppgaveInternalAliveMockImpl {
    @GetMapping
    public ResponseEntity isAlive() {
        return ResponseEntity.ok().build();
    }
}

File: vtp-pensjon-application/src/main/kotlin/no/nav/pensjon/vtp/mocks/oppgave/rest/OppgaveInternalAliveMockImpl.kt
package no.nav.pensjon.vtp.mocks.oppgave.rest

import io.swagger.v3.oas.annotations.tags.Tag
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@Tag(name = "Oppgave Mock")
@RequestMapping("/rest/oppgave/api/v1/internal/alive")
class OppgaveInternalAliveMockImpl {
    @GetMapping
    fun isAlive() = ResponseEntity.ok().build<Any>()
}
